source/common/res/features/popup-calculator/budget/budget_calc.css

```diff
@@ -1,8 +1,15 @@
+#toolkitPopupCalcInput  {
+    margin-top: .3em;
+    margin-bottom: .3em;
+    width: 176px;
+		text-align: right;
+}
+
 .budget-table-cell-activity {
   width: 17% !important;
 }
 
-.toolkit-popup-calc-button {
+.ytoolkit-popup-calc-button {
```

Comment:
Yeah, the y version should be deleted as that class is not needed in that css file.